Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ros:humble-ros-base-jammy
+
+# Install basic dev tools (And clean apt cache afterwards)
+RUN apt update \
+    && DEBIAN_FRONTEND=noninteractive \
+        apt -y --quiet --no-install-recommends install \
+        # Command-line editor
+        nano \
+        # Ping network tools
+        inetutils-ping \
+        # Install Lord IMU driver pkg
+        ros-$ROS_DISTRO-microstrain-inertial-driver \
```

Comment:
I flip-flopped about this a few times in the past, with pros/cons for pinning a specific version (In the past, we tended to pin specific versions _only_ iff no newer would do [since you assume no major interface breaks unless it's a major version change]). In fact recently (I think it was the spinnaker camera drivers) I was keen to get the newest driver asap because they were actively fixing an issue with the driver.

However, probably a better practice for big complex code-bases with multiple components [such as an AV stack] is to pin everything just in case, so I think this is how we should do it from now on. I'll add this pinned version (And we should do so from now on in all docker containers). I presume we should only pin hard dependencies like drivers & libraries, we can probably get away without pinning e.g. `nano` for the dev images.

Reply:
Agreed!